docs/release-notes/8.0/8.0.33-28.0.md

```diff
@@ -0,0 +1,47 @@
+# Percona XtraBackup 8.0.33-28 (2023-07-)
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
+
+This release includes improvements, features, and bug fixes. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+Percona XtraBackup 8.0.33-28 implements a new datasink that uses `FIFO` (named pipes). With the [FIFO datasink](..//..//xbcloud-binary-fifo-datasink.md) feature, you can benefit from faster backups by streaming data in parallel from XtraBackup to `xbcloud`. This feature is in [tech preview](..//..//glossary.md#tech-preview).
```

Comment:
We should clarify who will benefit from this feature. I feel that this statement claims that everyone would benefit from faster streaming.

We should clarify that it is intended for LAN usage who has S3 compatible storages like MinIO etc (marce has written about them in blog post).

Reply:
updated